### DISCUSSION TITLE
Multi-line label support

### DISCUSSION THREAD

**Discussion by daerath** [simonec73/threatsmanager#46] [Ideas]
Multi-line labels would be a nice feature to allow for more detail within the diagram. The only option now is to put that detail in the Description (e.g., a data store with label of "Job Profile" versus "Job Profile" and then on the second line "PII" or some other descriptive element). This would help with visual reviews by showing additional data on the diagram.

**Comment by simonec73** [comment]
A recent release has implemented Word wrap for labels (2.1.5).
